enclave/src/lib.rs

```diff
@@ -77,28 +77,40 @@ use substratee_worker_primitives::block::{
 };
 use substratee_worker_primitives::BlockHash;
 use utils::write_slice_and_whitespace_pad;
+
+use crate::constants::{
+    CALL_WORKER, OCEX_DEPOSIT, OCEX_MODULE, OCEX_RELEASE, OCEX_WITHDRAW, SHIELD_FUNDS,
+};
+use crate::utils::UnwrapOrSgxErrorUnexpected;
+
+use crate::extrinsic_deposit::PendingExtrinsicHelper;
+use serde::private::ser::serialize_tagged_newtype;
+use std::sync::atomic::{AtomicPtr, Ordering};
+
+static GLOBAL_POLKADEX_BALANCE_STORAGE: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
```

Comment:
Define the Nonce and pending extrinsic atomic pointer and related functions inside the extrinsic handler file

Reply:
done